man/adoc/bpftrace.adoc

```diff
@@ -434,9 +434,10 @@ inside a script config block.
 
 === Data Types
 
-The following fundamental integer types are provided by the language.
-Integers are internally represented as 64 bit signed.
-If you need another representation, you may cast to the following built in types:
+The following fundamental types are provided by the language.
+Note: Integers are by default represented as 64 bit signed but that can be 
+changed by either casting them or, for scratch variables, explicitly specifying 
+the type upon declaration.
```

Comment:
String is a fundamental data type too, right? Perhaps document that as well

Reply:
I'm going to update docs for type stuff in a follow up PR, there was some discussion in this PR or another (I can't recall).